cpp/environment/MatchEnv.cc

```diff
@@ -228,10 +235,9 @@ std::vector<vec3<float>> EnvDisjointSet::getAvgEnv(const unsigned int m)
 
     // loop through the vectors in env now, dividing by the total number
     // of contributing particle environments to make an average
-    for (unsigned int n = 0; n < m_max_num_neigh; n++)
+    for (auto& vec : env)
     {
-        vec3<float> normed = env[n] / static_cast<float>(N);
-        env[n] = normed;
+        vec = vec / static_cast<float>(N);
```

Comment:
Use vector operator overloads.

https://github.com/glotzerlab/freud/blob/dea4859079218a565918cc98d38143354571e01b/cpp/util/VectorMath.h#L237

```suggestion
        vec /= static_cast<float>(N);
```